internal/version/version.go

```diff
@@ -0,0 +1,177 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package version
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+var (
+	mon = monkit.Package()
+	// Timestamp is the UTC timestamp of the compilation time
+	Timestamp string
+	// CommitHash is the git hash of the code being compiled
+	CommitHash string
+	// Version is the semantic version set at compilation
+	// if not a valid semantic version Release should be false
+	Version = "v0.1.0"
+	// Release indicates whether the binary compiled is a release candidate
+	Release bool
+	// Build is a struct containing all relevant build information associated with the binary
+	Build Info
+)
+
+// Info is the versioning information for a binary
+type Info struct {
+	Timestamp  string `json:"timestamp,omitempty"`
```

Comment:
this should be a time.Time

Reply:
Cant we use an UNIX Timestamp better here?
Handling them in the protobuf is also easier i would suppose

Reply:
I am storing the Timestamp on the satellite as an INT64 as well. Makes comparison easier imo

Reply:
time.Time has a Unix method, so I'd still prefer a time.Time as the type here. if you want it to serialize to json as unix timestamp you can do:

```
type Info struct {
  Timestamp Time `json:"timestamp,omitempty"`
  ...
}

type Time time.Time

func (t Time) MarshalJSON() ([]byte, error) {
   return []byte(fmt.Sprint(t.Unix())), nil
}
```
to override json serialization